apps/prairielearn/src/migrations/20250822211535_enrollments__joined_at__add.test.sql

```diff
@@ -0,0 +1,16 @@
+-- BLOCK select_enrollment_for_user_in_course_instance
+SELECT
+  *
+FROM
+  enrollments
+WHERE
+  user_id = $user_id
+  AND course_instance_id = $course_instance_id;
+
+
+-- BLOCK old_ensure_enrollment
+INSERT INTO
```

Comment:
For example, the new query inserts a `joined_at` column, which isn't what I want.